lib/smart_answer_flows/coronavirus-find-support/outcomes/_paying_bills.erb

```diff
@@ -0,0 +1,45 @@
+## Paying bills
+
+### If you’re finding it hard to pay your rent, mortgage or bills
+
+[Find out if you can get help paying for childcare](https://www.gov.uk/get-childcare)
+
+[Find out if you can get help in your local area](https://www.gov.uk/coronavirus-local-help)
+
+<% if calculator.needs_help_in?("northern_ireland") %>
+  [Get help from Citizens Advice](https://www.citizensadvice.org.uk/about-us/northern-ireland/)
+
+  [Get help from the housing and debt helpline for Northern Ireland (Housing Rights)](https://www.housingrights.org.uk/)
+<% end %>
+
+<% if calculator.needs_help_in?("scotland") %>
+  [What to do if you cannot pay your rent or mortgage (Shelter Scotland)](https://scotland.shelter.org.uk/get_advice/scottish_housing_advice_coronavirus_COVID_19)
+
+  [What to do if you cannot pay your bills or mortgage, including if you cannot go out to pay them (Citizens Advice Scotland)](https://www.citizensadvice.org.uk/scotland/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/)
+<% end %>
+
+<% if calculator.needs_help_in?("england") %>
+  [What to do if you cannot pay your rent or mortgage (Shelter)](https://england.shelter.org.uk/housing_advice/coronavirus)
+
+  [What to do if you cannot pay your bills or mortgage, including if you cannot go out to pay them (Citizens Advice)](https://www.citizensadvice.org.uk/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/)
+<% end %>
+
+<% if calculator.needs_help_in?("wales") %>
+  [What to do if you cannot pay your rent or mortgage (Shelter Cymru)](https://sheltercymru.org.uk/get-advice/coronavirus/)
+
+  [What to do if you cannot pay your bills or mortgage, including if you cannot go out to pay them (Citizens Advice)](https://www.citizensadvice.org.uk/wales/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/)
+<% end %>
+
+#### Support and advice
```

Comment:
Might not be an issue - but h3 and h4 are styled the same for govspeak.

Reply:
I don't understand why would we do that? It doesn't make sense.  I guess if it is a problem we can make the H4 an H5 instead, assuming H5's aren't styled the same as H3 too!